scripts/send_slack_message.py

```diff
@@ -0,0 +1,40 @@
+import argparse
+import requests
+import json
+from github import Github
+
+from common import get_patch_version, env
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--token", type=str, required=True)
+    parser.add_argument("--webhook", type=str, required=True)
+    parser.add_argument("--slack_ids", type=str, required=True)
```

Comment:
Let's add a good description for `webhook` and `slack_ids` parameters. Otherwise, it's impossible to understand what format is expected